README.md

```diff
@@ -54,11 +54,6 @@
 - **[query and update animals table]**
 - **[query multiple tables]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- GETTING STARTED -->
 
```

Comment:
- [ ] I think 🤔 the information you added here is correct. This is a Vet clinic database project and not a Ruby on rails application, please let's add the correct information related to our project. ✍🏻 

![image](https://user-images.githubusercontent.com/79058364/235928233-df8f0016-613b-4758-9ded-2c50775d0fd7.png)